lib/pages/my_pixels/widgets/pixel_thumbnail.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class PixelThumbnail extends StatelessWidget {
```

Comment:
What do you think about a generic widget to render thumbnails and pixels? We can think about some parameters like width, height and source.